pgdrive/pg_config/collision_group.py

```diff
@@ -0,0 +1,38 @@
+from panda3d.bullet import BulletWorld
+
+
+class CollisionGroup:
+    Terrain = 2
+    Ego_Vehicle = 1
+    Lane_line = 3
+    Traffic_vehicle = 4
+    Lane_surface = 5
+
+    @classmethod
+    def set_collision_rule(cls, dynamic_world: BulletWorld):
+        # terrain collision
+        dynamic_world.setGroupCollisionFlag(cls.Terrain, cls.Terrain, False)
```

Comment:
could we use a loop to replace everything here?

Reply:
yes